Nodejs/Product/Nodejs/Workspace/ContextMenuProvider.cs

```diff
@@ -240,6 +295,14 @@ private void WriteNpmOutput(object sender, NpmLogEventArgs e)
                     this.outputPane.WriteLine(e.LogText.TrimEnd('\r', '\n'));
                 }
             }
+
+            private struct BuildProgressUpdater : IProgress<IFileContextActionProgressUpdate>
+            {
+                void IProgress<IFileContextActionProgressUpdate>.Report(IFileContextActionProgressUpdate value)
+                {
+                    // Should this be outputting? It doesn't seem be called
```

Comment:
Maybe add a assertion failure to understand when this happens.

Reply:
we have no where to output this too, updated comment